### ISSUE TITLE
Module parse failed: Unexpected token (2257:6)

### ISSUE THREAD

**Issue by gwiz123** [erikbrinkman/d3-dag#65]
App is unable to compile when using d3-dag version 0.8.1 with Create React App.
![image](https://user-images.githubusercontent.com/380239/127772921-91f4c5dd-3180-48cc-8b8e-3e8487ce217d.png)

This can be replicated by
- Creating a brand new Create React App "e.g. npx create-react-app my-app --template typescript"
- Import d3-dag into the App.tsx and call dagStratify() 

Attached example project 
[my-app.zip](https://github.com/erikbrinkman/d3-dag/files/6912762/my-app.zip)

**Comment by erikbrinkman** [comment]
When I ran your minimum example (I didn't download the zip), I got a different error pointing to an advanced language feature that wasn't supported. I had been hoping to move to later versions of javascript, but that seems configuration of tools like create-react-app are not set up to polyfill back down.

I just released 0.8.2 which targets ES6, so there shouldn't be any more issues like this, but since I didn't replicate your same error, and there are always challenges with bundling, I'm letting you know to see if this fixes the issue.
